index.js

```diff
@@ -1,17 +1,41 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+   this.items = [];
+   this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+  this.items.push(item);
+  this.length += 1;
+  this.items.sort(function (a,b){
+    return a-b;
+  });
+  }
+  
+  get(pos) {
+    if(pos > this.length){
```

Comment:
Aqui só lembra que se for passado um número negativo pode dar erro tb, então vc pode colocar ele junto nessa condição ;)